### PR TITLE
feat: support revised visor flags as part of lily changes

### PR DESCRIPTION
**Pull request by iand**
This is to support the changes made in https://github.com/filecoin-project/sentinel-visor/pull/464 which are expected to land in visor v0.6.0

### CODE REVIEW

**Comment by iand** [comment]
Diff handling in this PR is a bit confusing. The change is that database and lens flags have been removed from a section on their own to be repeated under the watch and walk commands.

File: charts/visor/templates/deployment.yaml [hunk 1]
```diff
@@ -55,8 +55,21 @@ spec:
           {{- if .Values.logLevelNamed }}
           - --log-level-named={{ .Values.logLevelNamed }}
           {{- end }}
+
```
Comment (placer14) [inline charts/visor/templates/deployment.yaml:58]: Nit: This will add whitespace to the values output and makes for noisy diffs in downstream processes. If we were separating logical groupings of options, then it might make sense. If we want whitespace within the template for organization sake, I'd prefer that it's within a comment block.

File: charts/visor/templates/deployment.yaml [hunk 2]
```diff
@@ -101,8 +100,28 @@ spec:
...
-          {{- with .Values.walk }}
+          - run
           - walk
+          {{- /*
+          # database args
+          */}}
+          {{- if .Values.pgPoolSize }}
+          - --db-pool-size={{ int .Values.pgPoolSize }}
+          {{- end }}
+
```
Comment (placer14) [inline charts/visor/templates/deployment.yaml:111]: Nit: unnecessary whitespace

**Review by placer14 — APPROVED** [review]
Minor nits. GTM.
